lib/copy/ncp.js

```diff
@@ -96,7 +96,7 @@ function ncp (source, dest, options, callback) {
 
   function copyFile (file, target) {
     var readStream = fs.createReadStream(file.name)
-    var writeStream = fs.createWriteStream(target, { mode: file.mode })
+    var writeStream = fs.createWriteStream(target)
```

Comment:
I'm concerned how this might affect *nix users with a non-standard umask. Is it possible that we could be writing a file with incorrect permissions that wouldn't allow us to do anything to the file? I haven't tested, just wondering.